sky/services/platform/interface.mojom

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module platform_interface;
+
+/// Specifies an acceptable orientation for the interface to be in
+/// A mask of acceptable interface orientations is created by or'ing individual
+/// options
+enum /* options */ InterfaceOrientation {
+  Portrait  = 1,
+  Landscape = 2,
+};
+
+/// Specifies a particular device orientation
+/// A mask of device interface orientations is created by or'ing individual
+/// options
+enum /* options*/ DeviceOrientation {
+  PortraitUp      = 1,
+  PortraitDown    = 2,
+  LandscapeLeft   = 4,
+  LandscapeRight  = 8,
+};
+
+/// Specifies how the status bar(s) should be displayed
+enum StatusBarsStyle {
+  /// The status bar(s) are fully visible
+  Visible,
+
+  /// The status bar on top of the interface is hidden
+  TopBarHidden,
+
+  /// All status bars and decorations are hidden
+  /// Synonymous with TopBarHidden on iOS. Maps to immersive mode on Android
+  AllHidden,
+};
+
+/// Controls specific aspects of the embedder interface
+interface Interface {
```

Comment:
`s/Interface/Chrome/`

Chrome is a term that refers to the decorations around the main content.